src/main/controllers/StepsToMakingYourClaimController.ts

```diff
@@ -6,13 +6,13 @@ import { FormContent } from '../definitions/form';
 import { AnyRecord } from '../definitions/util-types';
 
 import { getPageContent } from './helpers';
-
+let employeeStatus: string;
 const sections = [
   {
     title: (l: AnyRecord): string => l.section1.title,
     links: [
       {
-        url: PageUrls.DOB_DETAILS,
+        url: employeeStatus,
```

Comment:
This is in the wrong place. Should be below in place of PageUrls.PAST_EMPLOYER. This will cause the DOB page to break